README.md

```diff
@@ -41,3 +41,11 @@ We use `autopep8` and `isort` to automatically format the code to a common forma
 ### Linting and Testing
 
 We use `flake8` for linting, `mypy` for type checking and `pytest` for unit testing. To run these tools, just run `./lint_and_test.sh`.
+
+### Documentation
+
+We use the [Google docstring format](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) for documenting the code.
+We document every user-facing class or method, and enforce that using the flake8-docstrings library.
+
+The rendered documentation in `docs/docs.md` is then generated from the docstrings in the code using Sphinx and some heavy post-processing.
```

Comment:
This sentence sounds really weird. I think that this would be better:
"The documentation is then rendered from the docstrings in the code using Sphinx and some heavy post-processing and saved as `docs/docs.md`"